apps/wing/src/analytics/scripts/detached-export.ts

```diff
@@ -17,7 +26,7 @@ async function reportAnalytic() {
     throw new Error('No file analytic path provided');
   }
 
-  const analytics = new Analytics({ writeKey: 'sCqPF5xSscOjJdi5Tbkqu73vfF8zkZdw'});
+  const analytics = new Analytics({ writeKey: process.env.DEBUG === undefined ? segmentWriteKey : segmentDebugWriteKey});
```

Comment:
nit: I think it makes sense to invert this, so it matches the other DEBUG checks

```suggestion
  const analytics = new Analytics({ writeKey: process.env.DEBUG ? segmentDebugWriteKey : segmentWriteKey});
```